server/routes/orion.js

```diff
@@ -265,541 +175,390 @@ exports.getOrganizationRestaurants = function(req, res) {
+        utils.returnResponse(patchResponse, res);
+      })
+      .catch(function(err) {
+        utils.responseError(err, res);
+      });
+    } else {
+      utils.returnForbidden(res);
+    }
   })
   .catch(function(err) {
-    res.statusCode = err.statusCode;
-    res.json(err.error);
+    utils.responseError(err, res);
   });
 };
 
 exports.deleteReservation = function(req, res) {
-  utils.sendRequest('DELETE', null, req.params.id, req.headers)
-    .then(function(data) {
-      res.statusCode = data.statusCode;
-      res.end();
-    })
-    .catch(function(err) {
-      res.statusCode = err.statusCode;
-      res.json(err.error);
-    });
+  var reservationId = req.params.id;
+  utils.sendRequest('DELETE', null, reservationId, req.headers)
+  .then(function(deleteResponse) {
+    utils.returnResponse(deleteResponse, res);
+  })
+  .catch(function(err) {
+    utils.responseError(err, res);
+  });
 };
 
 exports.getReservations = function(req, res) {
   utils.getListByType('FoodEstablishmentReservation', null, req.headers)
-    .then(function(data) {
-      res.statusCode = data.statusCode;
-      res.json(utils.dataToSchema(data.body));
-    })
-    .catch(function(err) {
-      res.statusCode = err.statusCode;
-      res.end();
-    });
+  .then(function(reservations) {
+    utils.returnResponse(reservations, res);
+  })
+  .catch(function(err) {
+    utils.responseError(err, res);
+  });
 };
 
 exports.getUserReservations = function(req, res) {
-  var userReservations = [];
-  utils.getListByType('FoodEstablishmentReservation', null, req.headers)
-  .then(function(data) {
-    userReservations = utils.getUserReservations(
-      req.params.user,
-      data.body);
-    res.statusCode = data.statusCode;
-    res.json(utils.dataToSchema(userReservations));
+  var sqlList = [];
+  var queryString = {};
+  utils.addConditionToQuery(sqlList, 'underName', '==', req.params.user);
+  queryString.q = sqlList.join(';');
+  utils.getListByType('FoodEstablishmentReservation', null, req.headers,
+                      queryString)
+  .then(function(reservations) {
+    utils.returnResponse(reservations, res);
   })
   .catch(function(err) {
-    res.statusCode = err.statusCode;
-    res.end();
+    utils.responseError(err, res);
   });
 };
 
 exports.getRestaurantReservations = function(req, res) {
-  var restaurantReservations = [];
-  utils.getListByType('FoodEstablishmentReservation', null, req.headers)
-  .then(function(data) {
-    restaurantReservations = utils.getRestaurantReservations(
-      req.params.restaurant,
-      data.body);
-    res.statusCode = data.statusCode;
-    res.json(utils.dataToSchema(restaurantReservations));
+  var sqlList = [];
+  var queryString = {};
+  utils.addConditionToQuery(sqlList, 'reservationFor', '==',
+                            req.params.restaurant);
+  queryString.q = sqlList.join(';');
+  utils.getListByType('FoodEstablishmentReservation', null, req.headers,
+                      queryString)
```

Comment:
please be consistent with indentation, at some places you put the parameter which does not fit on the previous line just below the parenthesis above and in other places you do not do it. I would suggest all the code follows the same indentation convention, as here.